src/js/controller/instream-adapter.js

```diff
@@ -72,6 +74,8 @@ var InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
         // Make sure the original player's provider stops broadcasting events (pseudo-lock...)
         _controller.detachMedia();
 
+        this.trigger('adBreakStart', _getAdBreakData('adBreakStart'));
```

Comment:
The add plugins are calling `instream.init`. They should also trigger 'adBreakStart' at the time most appropriate to their implementation.

Reply:
@egreaves Please comment on this. It was a conclusion that we made after discussion.